# CompBioResourceCenter

Welcome to the Computational Biology Resources Read Me! In this curated collection, you will find a wealth of tools, databases, software packages, and resources tailored to support computational biologists and researchers in navigating the complexities of biological data analysis. 


## Free Cloud Computing Resources
[gitpod](https://www.gitpod.io) - Gitpod is a cloud development environment for teams to efficiently and securely develop software. \
[google colab](https://colab.research.google.com) - Colab is a hosted Jupyter Notebook service that requires no setup to use and provides free access to computing resources, including GPUs and TPUs. \
[kaggle](https://www.kaggle.com/) - Kaggle is the world's largest data science community with powerful tools and resources to help you achieve your data science goals. 

## Containers
[docker-builds](https://github.com/StaPH-B/docker-builds) - repository contains the Dockerfiles and other assorted files necessary for building Docker images for a variety of programs \
[seqera containers](https://seqera.io/containers/) - on-demand cointainers built 
## Pipeline Repos
[Awesome Nextflow](https://github.com/nextflow-io/awesome-nextflow) - A curated list of Nextflow pipelines \
[nf-core](https://nf-co.re/pipelines) - pipelines that are currently available as part of nf-core \
[IARC pipelines](https://github.com/IARCbioinfo/IARC-nf) -International Agency for Research on Cancer bioinformatics platform pipelines


## Target Identification 
[opentarget.org](https://www.opentargets.org/)- freely-available, open source tool that highlights variant-centric statistical evidence to allow both prioritisation of candidate causal variants at trait-associated loci, and identification of potential drug targets.

## CRISPR Technology 
[crispor](http://crispor.gi.ucsc.edu) - a program that helps design, evaluate and clone guide sequences for the CRISPR/Cas9 system \
[Tide](http://shinyapps.datacurators.nl/tide/) - For non-templated Cas9 editing \
[Tider](http://shinyapps.datacurators.nl/tider/) - For template-directed Cas9 editing

## Design of Experiments (DOE)

## Data Wrangling
[iSeq](https://github.com/BioOmics/iSeq?tab=readme-ov-file) -iSeq is a Bash script that allows you to download sequencing data and metadata from GSA, SRA, ENA, and DDBJ databases.

[Gwak](https://github.com/Kanaries/GWalkR?tab=readme-ov-file)- data visualization tool in R 
## Genomic/DNA 

## Transcriptomic/RNA
[DEBrowser](https://www.bioconductor.org/packages/release/bioc/vignettes/debrowser/inst/doc/DEBrowser.html) - Interactive Differential Expression Analysis Tool.

[ShinyGo](http://bioinformatics.sdstate.edu/go/)[GO analysis](https://github.com/evanpeikon/functional_enrichment_analysis)  
## Proteomic/Protein

[bolt] (https://github.com/jwohlwend/boltz) \
[chai](https://github.com/chaidiscovery/chai-lab) \
[alphafold](https://alphafold.ebi.ac.uk/) - AlphaFold DB provides open access to over 200 million protein structure predictions to accelerate scientific research. \
[colabfold](https://colab.research.google.com/github/sokrypton/ColabFold/blob/main/AlphaFold2.ipynb)- Coupled with Google Colaboratory, ColabFold becomes a free and accessible platform for protein folding \
[cfold](https://colab.research.google.com/github/patrickbryant1/Cfold/blob/master/Cfold.ipynb) -contains a pipeline for predicting alternative conformations of protein structures. \
[Foldseek](https://search.foldseek.com/search) - Server offers fast and sensitive protein structure alignments against large protein structure collections. \
[protein-design tools repo](https://design.adaptyvbio.com/tools) -adaptvbio repo of protein design tools \
[Protein-Ligand Binding Database](https://plbd.org/db/)- The PLBD contains thermodynamic and kinetic data describing proteinâ€‘small molecule compound (ligand) interactions. \
[nuerosnap repo](https://neurosnap.ai/services) - repo for list of protein folding tools \
[chaidiscovery](https://www.chaidiscovery.com/blog/introducing-chai-1)
[EvolvePro](https://github.com/mat10d/EvolvePro)
[DeepMutationalScanning]https://huggingface.co/spaces/thaidaev/zsp


[RoseTTAFold2NA](https://app.superbio.ai/apps/668c0b4c5a3d67435cdebf64)- RoseTTAFold2NA is an advanced computational tool developed to predict the structures of protein/nucleic acid complexes with high accuracy

[Chroma](https://github.com/generatebio/chroma)-chroma is a generative model for designing proteins programmatically.## Epigenomic/Epigenome
[UniFold](https://github.com/dptech-corp/Uni-Fold)-Alphafold in pytorch framework \
[ProteinX](https://github.com/bytedance/Protenix)-Alphafold3 in pytorch framework \
[PyMOLFold](https://github.com/colbyford/PyMOLfold)
# Protein Interaction
[Protein Interaction](https://pioneer.yulab.org/)- PIONEER: Generating whole-proteome human interactome using deep learning
## CADD
###ADMET
[SwissDrugSuite](http://www.swisstargetprediction.ch/) -suite of drug tools 
[admetlab](https://admetlab3.scbdd.com/)- admet data
[SyntheMol](https://github.com/swansonk14/SyntheMol?tab=readme-ov-file)- Generative AI for Drug Discovery
[RDKIT](https://www.rdkit.org/) \
[OECHEM](https://docs.eyesopen.com/toolkits/python/oechemtk/index.html) \
[polaris](https://polaris-hub.github.io/polaris/stable/) \
[DiffSBDD](https://github.com/arneschneuing/DiffSBDD)-Structure-based Drug Design with Equivariant Diffusion Models \
